renovate.json

```diff
@@ -0,0 +1,13 @@
+{
+	"extends": ["config:base", "group:all"],
+	"packageRules": [
+		{
+			"matchManagers": ["npm"],
```

Comment:
We also have gradle dependencies that have to be managed by Renovate.

Reply:
what about now?